homeassistant/components/lektrico/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The Lektrico Charging Station integration."""
+
+from __future__ import annotations
+
+from lektricowifi import Device
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_SERIAL_NUMBER, CONF_TYPE, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import LektricoDeviceDataUpdateCoordinator
+
+# List the platforms that charger supports.
+CHARGERS_PLATFORMS = [Platform.SENSOR]
+
+# List the platforms that load balancer device supports.
+LB_DEVICES_PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Lektrico Charging Station from a config entry."""
+    coordinator = LektricoDeviceDataUpdateCoordinator(
+        hass,
+        f"{entry.data[CONF_TYPE]}_{entry.data[ATTR_SERIAL_NUMBER]}",
+    )
+
+    await coordinator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```

Comment:
Let's use entry.runtime_data, there's a dev blog post out there, I'm on mobile so I don't have it at me

Reply:
I replaced hass.data with entry.runtime_data, thank you.